### DISCUSSION TITLE
aiohttp on a jailbroken iOS device

### DISCUSSION THREAD

**Discussion by Mahasvan** [aio-libs/aiohttp#5964] [Q&A] [answered]
So I've jailbroken my iPad Air 1, which runs iOS 12.5, and has python and pip installed using a Cydia tweak. Aiohttp refuses to be installed via `pip3 install aiohttp`. Has anyone tried this on their jailbroken iOS device? Is there some way that this is possible to install?

**Comment by webknjaz** [comment, marked as answer]
This is not something that we support. But I suppose if you have the whole build toolchain, it may be possible to compile it.

**Reply by Mahasvan** [reply]
Hmm I see, thanks for the answer!
